docs/reference/aggregations/bucket/geotilegrid-aggregation.asciidoc

```diff
@@ -132,7 +131,6 @@ POST /museums/_search?size=0
     }
 }
 --------------------------------------------------
-// CONSOLE
 // TEST[continued]
```

Comment:
@nik9000 

The build CI is throwing this error:

```
// TEST[continued] cannot be on first snippet in a file:
reference/aggregations/bucket/geotilegrid-aggregation.asciidoc[109:133](console)// TEST[continued]
```

For some reason, it's not recognizing the preceding snippet on line 33. Let me know what my next steps should be here.

I have a feeling this will break in other files too. This was just the first one the CI caught.

Reply:
Ignore. You answered this with your earlier comment.

> This'll need a change to the build infrastucture first. I can work on it tomorrow, hopefully.

Reply:
Yeah. I'll need to hack the build slightly to make this go. Not much though!